.babelrc

```diff
@@ -1,7 +1,6 @@
 {
   "presets": [
-    "latest",
-    "react"
+    "react-native"
```

Comment:
First thing I tried. You'll get a syntax error because the packager needs the react-native preset to package react-native.